Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+// This library defines the isPrBuild, prepareBuild and finalizeBuild methods
+@Library('jenkins.shared.library') _
+
+pipeline {
+  agent {
+    label 'ubuntu_docker_label'
+  }
+  stages {
+    stage("Setup") {
+      steps {
+        // prepareBuild is one of the Secure CICD helper methods
+        prepareBuild()
+      }
+    }
+    stage("Build Image") {
+      steps {
+           sh "make build"
+      }
+    }
+    stage("Push Image") {
+      //when { expression { ! isPrBuild() }  }
```

Comment:
did this check for `!isPrBuild()` not work?

Reply:
Thought that we'd want to push only from helm3 & master branch. But `!isPrBuild` also makes sense. Should I change to `!isPrBuild`?

Reply:
Please try !isPrBuild